README.md

```diff
@@ -425,12 +425,12 @@ $
 
 [TOC](#TOC)
 
-2) About preprocessor symbols **NDEBUG** and **DEBUG**
+2) About preprocessor symbols **NDEBUG** and **EXIV2_DEBUG_MESSAGES**
 
 In accordance with current practice, exiv2 respects the symbol *NDEBUG* which is set for Release builds.  There are sequences of code which are defined within:
 
 ```C++
-#ifdef DEBUG
+#ifdef EXIV2_DEBUG_MESSAGESDEBUG
```

Comment:
The preprocessor symbol name is wrong and it doesn't make sense given the next paragraph which statest that this is "not compiled for debug builds".

This paragraph should imho instead explain that this preprocessor symbol is for debug messages.